meteor/lib/api/prompter.ts

```diff
@@ -75,14 +75,14 @@ export namespace PrompterAPI {
 				parts: [],
 			}
 
-			_.each(parts, (part) => {
+			parts.forEach((part) => {
```

Comment:
changing this inner loop to a for-loop might have slight improvements in performance.

Reply:
Switched to a `for ... of` loop.